packages/portfolio-deploy/src/axelar-configs.js

```diff
@@ -187,6 +189,15 @@ const factoryAddresses = {
     Binance: '0x',
   },
 };
+/** @type {VaultAddressesMap} */
+const beefyVaults = {
```

Comment:
it's increasingly awkward to put contract address stuff in a file called `axelar-configs`.

I suggest `beefy-config.js` or `config-beefy.js`.